ui/src/index.ts

```diff
@@ -9,6 +9,22 @@ import {ErrorHandler} from 'app/components/error-handler';
 import {setupCustomValidators} from 'app/services/setup';
 import {AppRoutingComponent} from './app/routing/app-routing';
 
+import "../node_modules/@clr/icons/clr-icons.min.css";
```

Comment:
interesting, why do we have to specify the `node_modules` in the path to import the packages here?

Reply:
Likely there's something wrong with my configuration.